openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -119,23 +121,27 @@ def _imprint(self, tool, data):
 
     def _update_tool_with_data(self, tool, data):
         """Update tool node name and output path based on subset data"""
-        if "subset" not in data:
-            return
 
-        original_subset = tool.GetData("openpype.subset")
         subset = data["subset"]
-        if original_subset != subset:
-            # Subset change detected
-            # Update output filepath
-            workdir = os.path.normpath(legacy_io.Session["AVALON_WORKDIR"])
-            filename = f"{subset}..exr"
-            filepath = os.path.join(workdir, "render", subset, filename)
-            tool["Clip"] = filepath
-
-            # Rename tool
-            if tool.Name != subset:
-                print(f"Renaming {tool.Name} -> {subset}")
-                tool.SetAttrs({"TOOLS_Name": subset})
+        if subset is None:
+            self.log.warning("No subset found for _update_tool_with_data")
+            return
```

Comment:
I think **subset** cannot be `None` in this case and if it is, it _must_ be bug somewhere.  In that case it should probably raise exception or at least log error. But it might be something Fusion integration specific?

Reply:
Same question, how it can be `None`?